### ISSUE TITLE
Config issues and different views on app and browser

### ISSUE THREAD

**Issue by FlyHigh-Swe** [tmjo/charger-card#38]
Hi

I have issues configuration the card in both in the android app and using the web-interface. If I select the sensor and brand in the scroll list its not saved and I have to add it manually. Even if I select the brand (easee) I have to manually configure all the settings on the webinterface. If I use the android app I kind of get the opposite, the card is set up automatically but it only shows the default card even if I made some configurations.  

These screenshots are from the same view, the first is from the android app and the other from the web interface on the computer, The upper card is manually configured (mainly based on the example but with some changes), the lower card has minimal configuration (brand and sensor). 

Both will work fine, just strange that its different.
Android app:
![Screenshot_20221118-094820_Home Assistant](https://user-images.githubusercontent.com/62815879/202662426-6b70e430-2e63-42f2-aff1-7974af653035.jpg)
Desktop:
![Desktop](https://user-images.githubusercontent.com/62815879/202662489-3eef56ec-4092-4b5d-b505-0a88f8b20622.JPG)

This is my setup
Home Assistant 2022.11.3
Supervisor 2022.10.2
Operating System 9.3
RPI4

Thanks for a great card!

**Comment by tmjo** [comment]
Hi, yes, the UI config is broken and only manual yaml is functioning as it should. I'm working on it, hopefully not too far away.

**Comment by FlyHigh-Swe** [comment]
Do you have any idea why the cards act so different in the app and in the browser (sorry, I should probably have raised this as the main issue, let me know if its better if I should log a new issue).

And thanks for a great card!

**Comment by tmjo** [comment]
Hmm, I haven't seen that myself and I'm using the card for three different things. I guess there is something fishy in your config (could be caused by trying the UI config?) 

Can you post your manual config yaml, and I'll take a quick look?

**Comment by FlyHigh-Swe** [comment]
Here is the config for the upper card (that is correct on the web but the default eassee look on the app)

      - type: custom:charger-card
        entity: sensor.laddbox_status
        brand: easee
        show_leds: true
        details:
          name:
            entity_id: sensor.laddbox_status
            attribute: name
          location:
            entity_id: sensor.laddbox_status
            attribute: site_name
          status:
            entity_id: sensor.laddbox_status
          substatus:
            entity_id: sensor.laddbox_reason_for_no_current
          smartcharging:
            entity_id: switch.laddbox_smart_charging
          statetext:
            disconnected: disconnected
            awaiting_start: awaiting_start
            charging: charging
            completed: completed
            error: error
            ready_to_charge: ready_to_charge
          collapsiblebuttons:
            group1:
              text: Open Info
              icon: mdi:information
            group2:
              text: Open Settings
              icon: mdi:cog
            group3:
              text: Car status
              icon: mdi:car
          info_left:
            - entity_id: binary_sensor.laddbox_online
              text: online
            - service: null
              service_data:
                charger_id: EHXLTYLD
              text: update
              icon: mdi:file-download
              conditional_entity: binary_sensor.laddbox_update_available
          info_right:
            - entity_id: sensor.laddbox_voltage
              text: voltage
              unit_show: true
            - entity_id: sensor.laddbox_power
              text: power
              unit_show: true
          group1:
            - entity_id: sensor.laddbox_power
              text: power
              unit_show: true
            - entity_id: sensor.laddbox_current
              text: charger_current
              unit_show: true
            - entity_id: sensor.laddbox_circuit_current
              text: circuit_current
              unit_show: true
            - entity_id: sensor.laddbox_energy_per_hour
              text: energy_per_hour
              unit_show: true
            - entity_id: sensor.laddbox_session_energy
              text: session_energy
              unit_show: true
            - entity_id: sensor.laddbox_lifetime_energy
              text: lifetime_energy
              unit_show: true
          group2:
            - entity_id: switch.laddbox_is_enabled
              text: enabled
            - entity_id: switch.laddbox_enable_idle_current
              text: idle_current
            - entity_id: binary_sensor.laddbox_cable_locked
              text: cable_locked
            - entity_id: switch.laddbox_cable_locked_permanently
              text: Perm locked
            - entity_id: switch.laddbox_smart_charging
              text: smart_charging
            - entity_id: sensor.laddbox_cost_per_kwh
              text: cost_per_kwh
            - entity_id: binary_sensor.laddbox_update_available
              text: update_available
            - entity_id: binary_sensor.laddbox_basic_schedule
              text: schedule
          group3:
            - entity_id: input_text.waitingforcar
            - entity_id: sensor.distaance_to_car
              icon: mdi:sail-boat
          stats:
            default:
              - entity_id: sensor.laddbox_session_energy
                text: session_energy
                unit_show: true
              - entity_id: switch.laddbox_cable_locked_permanently
                text: cable_locked
              - entity_id: binary_sensor.laddbox_basic_schedule
                text: schedule
            disconnected:
              - entity_id: sensor.laddbox_session_energy
                text: session_energy
                unit_show: true
              - entity_id: switch.laddbox_cable_locked_permanently
                text: cable_locked
              - entity_id: calculated
                text: used_limit
                unit: A
                unit_show: true
                calc_function: min
                calc_entities:
                  - entity_id: sensor.laddbox_dynamic_charger_limit
                  - entity_id: sensor.laddbox_dynamic_circuit_limit
                  - entity_id: sensor.laddbox_max_charger_limit
                  - entity_id: sensor.laddbox_max_circuit_limit
                  - entity_id: sensor.laddbox_offline_circuit_limit
            awaiting_start:
              - entity_id: sensor.laddbox_session_energy
                text: session_energy
                unit_show: true
              - entity_id: binary_sensor.laddbox_basic_schedule
                text: schedule
              - entity_id: switch.laddbox_smart_charging
                text: smart_charging
              - entity_id: calculated
                text: used_limit
                unit: A
                unit_show: true
                calc_function: min
                calc_entities:
                  - entity_id: sensor.laddbox_dynamic_charger_limit
                  - entity_id: sensor.laddbox_dynamic_circuit_limit
                  - entity_id: sensor.laddbox_max_charger_limit
                  - entity_id: sensor.laddbox_max_circuit_limit
                  - entity_id: sensor.laddbox_offline_circuit_limit
            charging:
              - entity_id: sensor.laddbox_session_energy
                text: session_energy
                unit_show: true
              - entity_id: sensor.laddbox_energy_per_hour
                text: energy_per_hour
                unit_show: true
              - entity_id: sensor.laddbox_circuit_current
                text: circuit_current
                unit_show: true
              - entity_id: sensor.laddbox_output_limit
                text: output_limit
                unit_show: true
              - entity_id: sensor.laddbox_current
                text: current
                unit_show: true
              - entity_id: sensor.laddbox_power
                text: power
                unit_show: true
            completed:
              - entity_id: sensor.laddbox_session_energy
                text: session_energy
                unit_show: true
              - entity_id: binary_sensor.laddbox_basic_schedule
                text: schedule
              - entity_id: calculated
                text: used_limit
                unit: A
                unit_show: true
                calc_function: min
                calc_entities:
                  - entity_id: sensor.laddbox_dynamic_charger_limit
                  - entity_id: sensor.laddbox_dynamic_circuit_limit
                  - entity_id: sensor.laddbox_max_charger_limit
                  - entity_id: sensor.laddbox_max_circuit_limit
                  - entity_id: sensor.laddbox_offline_circuit_limit
            error:
              - entity_id: sensor.laddbox_session_energy
                text: session_energy
                unit_show: true
              - entity_id: binary_sensor.laddbox_basic_schedule
                text: schedule
            ready_to_charge:
              - entity_id: sensor.laddbox_session_energy
                text: session_energy
                unit_show: true
              - entity_id: binary_sensor.laddbox_basic_schedule
                text: schedule
              - entity_id: calculated
                text: used_limit
                unit: A
                unit_show: true
                calc_function: min
                calc_entities:
                  - entity_id: sensor.laddbox_dynamic_charger_limit
                  - entity_id: sensor.laddbox_dynamic_circuit_limit
                  - entity_id: sensor.laddbox_max_charger_limit
                  - entity_id: sensor.laddbox_max_circuit_limit
                  - entity_id: sensor.laddbox_offline_circuit_limit
          toolbar_left:
            default:
              - {}
            disconnected:
              - {}
            awaiting_start:
              - service: easee.stop
                service_data:
                  charger_id: EHXLTYLD
                text: stop
                icon: hass:stop
              - service: easee.resume
                service_data:
                  charger_id: EHXLTYLD
                text: resume
                icon: hass:play
              - service: easee.override_schedule
                service_data:
                  charger_id: EHXLTYLD
                text: override
                icon: hass:motion-play
            charging:
              - service: easee.stop
                service_data:
                  charger_id: EHXLTYLD
                text: stop
                icon: hass:stop
              - service: easee.pause
                service_data:
                  charger_id: EHXLTYLD
                text: pause
                icon: hass:pause
            completed:
              - service: easee.stop
                service_data:
                  charger_id: EHXLTYLD
                text: stop
                icon: hass:stop
              - service: easee.override_schedule
                service_data:
                  charger_id: EHXLTYLD
                text: override
                icon: hass:motion-play
            error:
              - service: easee.reboot
                service_data:
                  charger_id: EHXLTYLD
                text: reboot
                icon: hass:restart
            ready_to_charge:
              - service: easee.stop
                service_data:
                  charger_id: EHXLTYLD
                text: stop
                icon: hass:stop
              - service: easee.override_schedule
                service_data:
                  charger_id: EHXLTYLD
                text: override
                icon: hass:motion-play
          toolbar_right:
            default:
              - {}



and this is the config for the lower card (that works in the app but not on the web)


      - type: custom:charger-card
        entity: sensor.laddbox_status
        brand: easee


I tried both chrome and edge on the computer and it looks the same, first card looks as expected, second dont work.
If its only me I might have an issue with a card mod or other add on.
Worst case is that I simply build the default view manually in yaml so it looks the same regardless (web uses the yaml code and app the default layout for the card).

**Comment by tmjo** [comment]
I couldn't see anything wrong while looking briefly. The second part shouldn't really work, because what is supposed to happen when just giving `brand: easee` is that the UI config has a generator that is supposed to create the rest of the code based on my 'templates' to give a default setup. The small config should basically only show the card with no data except the main status, so corresponding to your second desktop image. 

 Could it be that there is some caching issue or something? I've also got two cards, one below the other, that works fine.

**Comment by FlyHigh-Swe** [comment]
You are absolutely right,, I had to empty the cache on the phone. I had tried to force a stop, reload resources and even rebooted the phone but it was not until I manually emptied the cache for the app that I got the same look as on the web.
Since I recreated the card a few times and my wife had similar issues I never thought of this.

**Comment by FlyHigh-Swe** [comment]
All that was needed was to empty the cache on the phone app manually

**Comment by tmjo** [comment]
Glad you got it solved!
